memberships/templates/base.html

```diff
@@ -23,7 +25,8 @@
 
 
         <nav class="navbar navbar-expand-lg">
-          <h1><a href="/" title="Home">Geek.Zone</a></h1>
+          <!-- <h1 class="logo"><a href="/" title="Home">Geek.Zone</a><span></span></h1> -->
+          <img class="logo" src="https://geek.zone/dl21?display" alt="">
```

Comment:
Could we make this a link to `/`? Also could we add some alt text of "geekzone logo" or something? Just in case any of our users are using a screen reader.

Reply:
yes, totally forgot to remove the uncommented line below and link the the logo but will do in the next iteration